src/pages/About.js

```diff
@@ -4,59 +4,60 @@ import { NavLink } from 'react-router-dom'
 import { motion } from 'framer-motion'
 import { pageAnimation } from '../animation'
 
-export default function About () {
-    return (
-        <motion.div className="about" exit="exit" variants={pageAnimation} initial="hidden" animate="show">
-
-            <div className="about-the-author">
-
-                <div className="author-image">
-                    <img src={flaterrol} alt="author"/>
-                </div>
-
-                <div className="author-bio">
-                    <h1>About the Creator</h1>
-                    <p>
-                    &nbsp;&nbsp;My name is Errol Watson. I'm currently based in Baltimore, Maryland. I am the sole developer of this site.
-                    I got into programming after building my first gaming PC.
-                    The build was relatively easy and mostly anticlimactic. Thusly, after it was up and running,
-                    I became more curious about how all the pieces in my computer and its applications worked.
-                    I'm grateful for finding programming because I am the kind of person who enjoys deep dives into subjects,&nbsp;
-                    and programming is <span style={{"font-style": "italic"}}>deep.</span>&nbsp;
```

Comment:
This span tag could be replaced with `<em>deep</em>`

You could also create a utility selector in your app.scss file that looks like `.italic { font-size: italic }` and then apply it as `<span className="italic">`

Or you can do it this way. There are times when it's perfectly fine to keep it inline.